sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs

```diff
@@ -498,14 +498,16 @@ public virtual async Task CloseAsync(CancellationToken cancellationToken = defau
         ///
         /// <returns>A task to be resolved on when the operation has completed.</returns>
         /// <exception cref="ServiceBusException">
-        ///   The lock for the message has expired or the message has already been completed. This does not apply for session-enabled
-        ///   entities.
-        ///   The <see cref="ServiceBusException.Reason" /> will be set to <see cref="ServiceBusFailureReason.MessageLockLost"/> in this case.
-        /// </exception>
-        /// <exception cref="ServiceBusException">
-        ///   The lock for the session has expired or the message has already been completed. This only applies for session-enabled
-        ///   entities.
-        ///   The <see cref="ServiceBusException.Reason" /> will be set to <see cref="ServiceBusFailureReason.SessionLockLost"/> in this case.
+        ///   <list type="bullet">
+        ///     <item>
+        ///       The lock for the message has expired or the message has already been completed. This does not apply for session-enabled entities.
+        ///       The <see cref="ServiceBusException.Reason" /> will be set to <see cref="ServiceBusFailureReason.MessageLockLost"/> in this case.
+        ///     </item>
```

Comment:
```suggestion
        ///     <item>
        ///        <description>
        ///          The lock for the message has expired or the message has already been completed. This does not apply for session-enabled entities.
        ///          The <see cref="ServiceBusException.Reason" /> will be set to <see cref="ServiceBusFailureReason.MessageLockLost"/> in this case.
        ///        </description> 
        ///     </item>
```

There needs to be a `<description>` element wrapping the text for a list tiem to be valid.   MS Docs won't render these if the `<description>` is missing.    _([ref](https://docs.microsoft.com/dotnet/csharp/language-reference/language-specification/documentation-comments#d37-list))_